WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewController.swift

```diff
@@ -1,17 +1,27 @@
 import SwiftUI
 import UIKit
+import Yosemite
 
 /// Displays a grid view of all available menu in the "Menu" tab (eg. View Store, Reviews, Coupons, etc...)
 final class HubMenuViewController: UIHostingController<HubMenu> {
+    private var hubMenu: HubMenu
+
     init(siteID: Int64, navigationController: UINavigationController?) {
-        super.init(rootView: HubMenu(siteID: siteID, navigationController: navigationController))
+        hubMenu = HubMenu(siteID: siteID, navigationController: navigationController)
+        super.init(rootView: hubMenu)
         configureNavigationBar()
         configureTabBarItem()
     }
 
     required dynamic init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    /// Present the specific Review Details View from a push notification
+    ///
+    func pushReviewDetailsViewController(using parcel: ProductReviewFromNoteParcel) {
+        hubMenu.pushReviewDetailsView(using: parcel)
```

Comment:
nit: I wonder if we can just call `rootView` here so that we don't need to keep a reference to `HubMenu` in init?

```suggestion
        rootView.pushReviewDetailsView(using: parcel)
```

Reply:
Great suggestion, I didn't know that the property `rootView` was available here 💯